GAME/TYPES.H

```diff
@@ -1,6 +1,9 @@
 #ifndef TYPES_H
 #define TYPES_H
 
+// required for NULL macro because if we define it ourselves it shits brixes
+#include <stdlib.h>
```

Comment:
Probably better to include in individual C files and not keep it global to ensure code is more usable standalone.

Reply:
Not sure what you mean
Do you mean that the `#include <stdlib.h>` statement should be put directly in the files that make use of NULL et al, instead of including it once globally?